test/h3core.spec.js

```diff
@@ -1405,102 +1367,102 @@ test('experimentalH3ToLocalIj / experimentalLocalIjToH3', assert => {
         ['8828308289fffff', {i: 393, j: 336}]
     ].forEach(([h3Index, coords]) => {
         assert.deepEqual(
-            h3.experimentalH3ToLocalIj(origin, h3Index),
+            h3.cellToLocalIj(origin, h3Index),
             coords,
             `Got expected coordinates for ${h3Index}`
         );
         assert.deepEqual(
-            h3.experimentalLocalIjToH3(origin, coords),
+            h3.localIjToCell(origin, coords),
             h3Index,
             `Got expected H3 index for ${JSON.stringify(coords)}`
         );
     });
     assert.end();
 });
 
-test('experimentalH3ToLocalIj / experimentalLocalIjToH3 - Pentagon', assert => {
+test('cellToLocalIj / localIjToCell - Pentagon', assert => {
     const origin = '811c3ffffffffff';
     [
         [origin, {i: 0, j: 0}],
         ['811d3ffffffffff', {i: 1, j: 0}],
         ['811cfffffffffff', {i: -1, j: 0}]
     ].forEach(([h3Index, coords]) => {
         assert.deepEqual(
-            h3.experimentalH3ToLocalIj(origin, h3Index),
+            h3.cellToLocalIj(origin, h3Index),
             coords,
             `Got expected coordinates for ${h3Index}`
         );
         assert.deepEqual(
-            h3.experimentalLocalIjToH3(origin, coords),
+            h3.localIjToCell(origin, coords),
             h3Index,
             `Got expected H3 index for ${JSON.stringify(coords)}`
         );
     });
     assert.end();
 });
 
-test('experimentalH3ToLocalIj - errors', assert => {
+test('cellToLocalIj - errors', assert => {
     assert.throws(
-        () => h3.experimentalH3ToLocalIj('832830fffffffff', '822837fffffffff'),
-        /Incompatible/,
+        () => h3.cellToLocalIj('832830fffffffff', '822837fffffffff'),
+        /\(12\)$/,
         'Got expected error'
     );
     assert.throws(
-        () => h3.experimentalH3ToLocalIj('822a17fffffffff', '822837fffffffff'),
-        /too far/,
+        () => h3.cellToLocalIj('822a17fffffffff', '822837fffffffff'),
+        /\(1\)$/,
         'Got expected error'
     );
     assert.throws(
-        () => h3.experimentalH3ToLocalIj('8828308281fffff', '8841492553fffff'),
-        /too far/,
+        () => h3.cellToLocalIj('8828308281fffff', '8841492553fffff'),
+        /\(1\)$/,
         'Got expected error for opposite sides of the world'
     );
     assert.throws(
-        () => h3.experimentalH3ToLocalIj('81283ffffffffff', '811cbffffffffff'),
-        /pentagon distortion/,
+        () => h3.cellToLocalIj('81283ffffffffff', '811cbffffffffff'),
+        /\(1\)$/,
         'Got expected error'
     );
     assert.throws(
-        () => h3.experimentalH3ToLocalIj('811d3ffffffffff', '8122bffffffffff'),
-        /pentagon distortion/,
+        () => h3.cellToLocalIj('811d3ffffffffff', '8122bffffffffff'),
+        /\(1\)$/,
```

Comment:
For my info, why did we have two distinct errors before and now we only have one generic one?

Reply:
I believe before we were differentiating some different codes based on where they were returned in the code, and then figuring out the text from that. Now both code sites are considered "E_FAILED" without more detail.